src/head.rs

```diff
@@ -0,0 +1,27 @@
+use bevy::{math::DVec2, prelude::*, utils::Uuid};
+
+use bevy_vello::prelude::*;
+
+#[derive(Default, Copy, Clone, PartialEq, Eq, Hash)]
+pub struct ShapeId(Uuid);
+
+#[derive(Component, Default, Copy, Clone)]
+pub struct Head {
+    pub shape_id: ShapeId,
+
+    pub scale: f32,
+    pub offset: f32,
+    pub rotation_offset: f32,
+}
+
+#[derive(Resource)]
+pub struct Shapes {
+    pub scenes: std::collections::HashMap<ShapeId, &'static mut vello::Scene>,
+}
+
+pub trait VectorBorder {
+    /// translation of the chosen "apex"
```

Comment:
```suggestion
    /// Translation of the border at a specific `time` value.
```
Just a suggestion on having the first letter of a comment sentence to be capital cuz it's going into documentation anyways. (Would be nice if this convention is applied to all comments as well. Full stop is not necessary on non-doc comments.)

I am also curious what does **_"apex"_** here means haha

Reply:
> Just a suggestion on having the first letter of a comment sentence to be capital

oops yeah sorry

> I am also curious what does "apex" here means haha

i intended "apex" to mean exactly what you've suggested haha.